test/Interop/Cxx/members/emit-method-from-called-method-of-nested-struct.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
+//
+// REQUIRES: executable_test
+
+import MethodCallsMethodFromNestedStruct
+import StdlibUnittest
+
+var MembersTestSuite = TestSuite("MembersTestSuite")
+
+MembersTestSuite.test("emit-method-from-method-of-nested-struct") {
```

Comment:
Nit: if you'd rather, you can use spaces here.

Reply:
Done. Thanks!